src/effects/builtin/amplify/AmplifyView.qml

```diff
@@ -45,12 +45,20 @@ EffectBase {
                 text: qsTrc("effects/amplify", "Amplification (dB):")
             }
 
+            Binding {
```

Comment:
I'd add some comment explaining why we need this magic - I'm sure I will not remember that the next time I look at this code :)

Reply:
Done! Thank you!